### ISSUE TITLE
Add landscape and portrait options in the Android version.

### ISSUE THREAD

**Issue by Rinaex** [Admiral-Billy/Pokerogue-App#87]
I hope to be able to control the screen orientation through the app, rather than having my phone control it.

**Comment by Admiral-Billy** [comment]
Is there any particular reason you're wanting this? It doesn't seem like a very big inconvenience to just rotate your phone and turn on rotation lock if playing in bed on your side or whatever is a concern. There could be something I'm forgetting though, but I just want to make sure as any UI stuff I add to the app is probably going to be pretty difficult, especially because the app is fullscreen and I wouldn't want to reduce the game's screen size further.

**Comment by ZehWallace** [comment]
Isn't it just a setting that to set the default of an android app to landscape? I think you only need to set it in the AndroidManifest.xml file for generating the apk. 

https://stackoverflow.com/questions/14182366/making-an-android-app-run-full-screen-and-landscape

**Comment by Rinaex** [comment]
> Is there any particular reason you're wanting this? It doesn't seem like a very big inconvenience to just rotate your phone and turn on rotation lock if playing in bed on your side or whatever is a concern. There could be something I'm forgetting though, but I just want to make sure as any UI stuff I add to the app is probably going to be pretty difficult, especially because the app is fullscreen and I wouldn't want to reduce the game's screen size further.

Yes, normally there's no inconvenience in turning on the rotary lock, but from time to time I'll switch windows to chat with my friends while playing and I'll need to lock the screen to portrait once again.

Maybe you could consider adding a floating button menu with some functions like refresh, switch screen orientation, open tools page and so on!

**Comment by Admiral-Billy** [comment]
If I ever do add a little menu similar to the desktop version, I'll look into it then.

**Comment by AstronW** [comment]
> If I ever do add a little menu similar to the desktop version, I'll look into it then.

I also have the same problem. I hope to add an option to select landscape or portrait when selecting online or offline mode.
